src/app/UserLogin.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledLabel = styled.label`
+  display: grid;
+  padding-bottom: 5px;
+  font-size: 0.6em;
```

Comment:
Is there a special reason why you are using "em" and not "rem" here? "em" takes the font-size of the parent-element.

Reply:
Thanks! I missed that. I should go through and check whether I've consistently used rem...